_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -129,6 +129,9 @@ static void FillSurfaceAttrs(std::vector<VASurfaceAttrib> &attrib, unsigned int
             attrib[0].value.value.i  = VA_FOURCC_P208;
             format                   = VA_FOURCC_P208;
             break;
+        case MFX_FOURCC_P010:
+            format = VA_RT_FORMAT_YUV420_10;
```

Comment:
Do we need to add VA API guard here using VA_CHECK_VERSION?

Reply:
> Do we need to add VA API guard here using VA_CHECK_VERSION?

We have not such guards in other places with MFX_FOURCC_P010

Reply:
I meant guard for VA_RT_FORMAT_YUV420_10 (see mfx_h265_encode_vaapi.cpp, VAAPIEncoder::CreateAuxilliaryDevice())